content/en/news/support/announcing-1.4-eol/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Support for Istio 1.4 ends on June 5th, 2020
+subtitle: Support Announcement
+description: Upcoming Istio 1.4 end of life announcement.
+publishdate: 2020-05-05
+aliases:
+    - /news/2020/announcing-1.4-eol
+---
+
+According to Istio's [support policy](/about/release-cadence/), LTS releases like 1.4 are supported for three months after the next LTS release. Since [1.5 was released on March 5th](/news/releases/1.5.x/announcing-1.5/), support for 1.4 will end on June 5th, 2020.
+
+At that point we will stop back-porting fixes for security issues and critical bugs to 1.4, so we encourage you to upgrade to the latest version of Istio ({{<istio_release_name>}}).  If you don't do this you may put yourself in the position of having to do a major upgrade on a short timeframe to pick up a critical fix.
```

Comment:
Just to check: this renders as "we encourage you to upgrade to the latest version of Istio (1.6)". Do we encourage everyone to upgrade to 1.6 at this point?

If so, then fine to leave alone; I just wasn't sure.

Reply:
When we cherrypick to the 1.5 branch (current istio.io) it will show 1.5 I believe. Then once 1.6 launches it will be 1.6,which I think is expected behavior

Reply:
Great--thanks!

Reply:
If you look at the various revisions of the site, preliminary, istio, and the archives (the later aren't rebuilt), you'll see that preliminary points to the next unreleased version (today 1.6.0) so all the unsupported pages show 1.6.0. istio, points to the latest related version, so they all say 1.5.2. The archives are frozen to the value when they were last built. I see it's 1.4.6 in the 1.4 archives.